presto-product-tests/src/main/java/com/facebook/presto/tests/TestGroups.java

```diff
@@ -47,6 +47,7 @@
     public static final String HDFS_IMPERSONATION = "hdfs_impersonation";
     public static final String HDFS_NO_IMPERSONATION = "hdfs_no_impersonation";
     public static final String AUTHORIZATION = "authorization";
+    public static final String HIVE_1_1_0 = "hive_1_1_0";
```

Comment:
I think it might be more appropriate to call it 'pre-hive_1_1'? Otherwise it seems like the tests are *only* for Hive 1.1.0.